org-journal.el

```diff
@@ -467,7 +467,8 @@ before it will be deposed."
      (with-current-buffer buf
        (unless buffer-exists
          (insert-file-contents ,file))
-       (setq result (progn ,@body)))
+       (setq result (let ((major-mode 'org-mode))
```

Comment:
Please add a comment describing the reason for this, and mentioning that it does not activate org-mode.